graphs/bgp/phase_king.go

```diff
@@ -0,0 +1,229 @@
+package phase_king
```

Comment:
General comments about naming convention:
1. if we stick by your comments it should be rather a directory (and within all test names) "consensus" not "bgp",
2. "phase_king" is fine, as long as its queen counterpart would be referred consistently as "phase_queen", otherwise "king" and "queen" make more sense - please consider it in advance.